{{ cookiecutter.project_slug }}/pyproject.toml

```
@@ -36,17 +36,17 @@ dependencies = [
 [project.optional-dependencies]
 doc = [
     "invoke ~=2.2",
-    "mike ~=1.1",
+    "mike >=1.1.0,<3",
```

```suggestion
    "mike ~=2.0",
```
Let's just jump straight to only using v2.